### ISSUE TITLE
No example of non-positional record construction in documentation

### ISSUE THREAD

**Issue by gergoerdi** [idris-lang/Idris2#2534]
https://idris2.readthedocs.io/en/latest/tutorial/typesfuns.html#records doesn't show the syntax for constructing a record with non-positional arguments:

```
fred : Person
fred = MkPerson "Fred" "Joe" "Bloggs" 30
```

Luckily, I've found https://github.com/idris-lang/Idris2/pull/607#issuecomment-682456667 which shows that one can also write:

```
fred : Person
fred = MkPerson{ firstName = "Fred", middleName = "Joe", lastName = "Bloggs", age = 30 }
```

**Comment by buzden** [comment]
> https://idris2.readthedocs.io/en/latest/tutorial/typesfuns.html#records doesn't show the syntax for constructing a record with non-positional arguments

Right. I think it is because historically it's documented [here](https://idris2.readthedocs.io/en/latest/updates/updates.html#function-application-syntax-additions) and because record's constructor is just a usual function from this perspective. Maybe, at least a link should be added.
